web/src/containers/BudgetSummary.js

```diff
@@ -178,11 +159,13 @@ const BudgetSummary = ({ activities, data, years }) => (
           return (
             <tr
               key={ffy}
-              className={ffy === "total" && "budget-table--row__highlight budget-table--total"}
```

Comment:
This results in `className` being either `"budget-table--row__highlight budget-table--total"` or `false`. It turns out that React shouts at you if your `className` isn't a string. Who knew?

Reply:
Are you saying that the component... has no class? ;-)